### PR TITLE
[Impeller] switch fallback default from ImpellerVulkan to SkiaOpenGLES

### PR DESCRIPTION
**Pull request by jonahwilliams**
Unblocks shipping Impeller on Android before GLES is ready.

### CODE REVIEW

**Review by matanlurey — APPROVED** [review]

File: shell/platform/android/platform_view_android_unittests.cc [hunk 1]
```diff
@@ -22,7 +22,7 @@ TEST(AndroidPlatformView, SelectsVulkanBasedOnApiLevel) {
               AndroidRenderingAPI::kImpellerVulkan);
   } else {
     EXPECT_EQ(FlutterMain::SelectedRenderingAPI(settings),
-              AndroidRenderingAPI::kImpellerOpenGLES);
+              AndroidRenderingAPI::kSkiaOpenGLES);
```
Comment (zanderso) [inline shell/platform/android/platform_view_android_unittests.cc:25]: Not familiar with how these tests are run, but curious how we make sure both of these branches are covered in CI.

**Review by zanderso — APPROVED** [review]
lgtm w/ question

File: shell/platform/android/platform_view_android_unittests.cc (hunk 1 above)
Comment (jonahwilliams) [reply shell/platform/android/platform_view_android_unittests.cc:25]: Good question! I have no idea :)

Comment (jonahwilliams) [reply shell/platform/android/platform_view_android_unittests.cc:25]: It looks like in https://github.com/flutter/engine/blob/main/ci/builders/linux_android_emulator.json we have API 28 and unspecified (which might be 33)

**Pull request merged**